WorkflowSubmission/Program.cs

```diff
@@ -32,11 +32,11 @@ async static Task Main(string[] args)
             // Submit files to Workflow
             WorkflowSubmission workflowSubmission = new WorkflowSubmission(client)
             {
-                Id = workflow.Id,
+                WorkflowId = workflow.Id,
                 Files = FILE_LIST
```

Comment:
If a user wants to use streams could they do something like:
`WorkflowSubmission workflowSubmission = new WorkflowSubmission(client)
￼            {
￼                Id = workflow.Id,
￼                WorkflowId = workflow.Id,
￼                Steams = STREAM_LIST`

Reply:
Yeah, steams and urls both can be used instead